pkg/chain/event.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// TODO: Rename to `KeepCreatedEvent`
```

Comment:
I believe we need to stick to the name already defined in the `BondedECDSAKeepFactory` contract.